cluster-scope/base/core/namespaces/datahub-linkedin/namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: datahub-linkedin
+    annotations:
+        openshift.io/requester: datahub-linkedin
+        openshift.io/display-name: "DataHub: The Metadata Platform for the Modern Data Stack"
+        op1st/project-owner: operate-first
+        op1st/onboarding-issue: "https://github.com/os-climate/os_c_data_commons/issues/807"
```

Comment:
issue link doesn't seem to work, maybe it should be: https://github.com/os-climate/os_c_data_commons/issues/163 ?